recipes/click-help-colors/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "click-help-colors" %}
+{% set version = "0.3" %}
+{% set sha256 = "dd1f2de37ab30a127314423c0e20346632b7c42eb562e9afc34005635c04563f" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - click
+
+  run:
+    - python
+    - click
+
+test:
+  imports:
+    - click_help_colors
+
+about:
+  home: https://github.com/r-m-n/click-help-colors
+  license: MIT
+  license_file: LICENSE.txt
```

Comment:
The `LICENSE.txt` file is not included in the package on pypi.io since the project is missing a `MANIFEST.in` file. Comment out this line and add a comment to add this later once it appears upstream. You might also raise an issue on the issue tracker for the project. This is what is causing the build to fail currently.